docs/UserGuide.md

```diff
@@ -18,7 +43,7 @@ Expense Expert is a **desktop app for keeping track of your expenses, and managi
 1. Copy the file to the folder you want to use as the _home folder_ for your Expense Expert.
 
 1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+   ![Ui](images/ApplicationStart.png)
```

Comment:
I think Ui.png is the standard name that is not supposed to be changed

Reply:
Updated.